### DISCUSSION TITLE
p/971404f2f72c429083a614f8ee522237/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#6] [Announcements]
# p/971404f2f72c429083a614f8ee522237/

年初计划  总结似乎很多人已经发布了 2021 年的年终总结，吾辈现在好像有点晚了？不过权当新年寄语吧！ 首先祝泥萌新年快乐！  今年发生了一些特别的事情  更换工作，并且从广州跑到了北京 基于 web 技术实现插件体系 在生产中验证了许多技术的可行性 kigurumi 穿了全套的 cosplay 服装  也发一下今年的 github 贡献图  更换工作，并且从广州跑到了北京毫无疑问，今年对吾辈

https://blog.rxliuli.com/p/971404f2f72c429083a614f8ee522237/

**Comment by Wshengyi** [comment]
不知道在临近2022年底，你是否做到

**Reply by rxliuli** [reply]
现在的大致情况是这样，但重要目标发生了一些变化

具体来讲

- 补足和实践理论层面的知识 -- 已经看完 sicpjs，花了大约两个月左右
- 继续维护和改进 liuli-tools/liuli-utils -- 这块还在维护，最近刚迁移 esm
- 继续维护 joplin-utils 系列的笔记相关工具 -- joplin-vscode-plugin 发布 1.0，目前安装数量接近 10k
- 提高发布博客的质量，而不是简单的笔记映射 -- 好吧，发布的 blog 确实变少了，一方面，学习日语占了一部分时间。另一方面，也想写一些更复杂的内容。
- 参与一些知名的开源项目并提 PR，而非仅仅做自己的东西 -- 好吧，提了一些小的 PR，今年大概只有十几个

生活上希望更加自律和保持精神，希望下面几件事情都有一些进展

- 保持作息规律，日常水果和牛奶补充营养 -– 实践的还算不错
- 开始一些简单的锻炼，例如跑步 –- 换成了早上玩 30~60m 的节奏光剑（VR 游戏）
- 尝试更多工作之外基于现实的社交活动 –- 没有进展
- kigurumi 出一些室外日常照片 -- 没有太多进展，但之前一个网友来帮忙拍过几张照骗
